### PR TITLE
Fix WebIO deprecations, add Project.toml.

### PR DESCRIPTION
**Pull request by twavv**

### CODE REVIEW

**Comment by twavv** [comment]
This fixes some style things in the JS, some deprecations, and moves to Project.toml instead of REQUIRE.

File: Project.toml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+name = "Knockout"
+uuid = "bcebb21b-c2e3-54f8-a781-646b90f6d2cc"
+license = "MIT"
+version = "0.8.3"
```
Comment (piever) [inline Project.toml]: Current release is 0.2.2 so I imagine this should be 0.2.3?

**Comment by piever** [comment]
Thanks!

File: Project.toml (hunk 1 above)
Comment (twavv) [reply Project.toml]: Yes! I got the minor from WebIO and the patch from Knockout... 🌮

**Pull request merged**